basics/uri.adoc

```diff
@@ -417,7 +417,7 @@ But not these:
 [example]
 ----
 //vcu.other.device/0/3/1     // non-local authority
-/1/3/3                       // wrong resource
+/0/3/3                       // wrong resource
```

Comment:
Are there any reserved ue_ids? Is zero kosher to use? If so, that's fine, change just caught my eye.

Reply:
@PLeVasseur I'm formulating the registry topic shortly (for the lower 16 bits). For Instance id, we use 0 to mean "default/not set" and 0xFFFF to mean wildcard but any other value is permitted.